src/v2/Apps/Fair/Components/FairEditorial/FairEditorialItem.tsx

```diff
@@ -39,27 +55,55 @@ export const FairEditorialItem: React.FC<FairEditorialItemProps> = ({
     action: ActionType.clickedArticleGroup,
   }
 
+  const image = article.thumbnailImage?.[size]!
+  const variants = {
+    large: {
+      primary: ["xl", "xxl"] as TextVariant[],
+    },
+    small: {
+      primary: ["lg", "lg", "xl"] as TextVariant[],
```

Comment:
question (non-blocking): I haven't used the `variant` property of a Text component before, so I am curious what this array means.

Reply:
That might be used if you want to show different text sizes depending on screen width